documentation/APIbacklog.md

```diff
@@ -662,6 +699,26 @@ This is a live doc that captures the status of all the APIs which have been form
       </td>
       <td>No release</td>
     </tr>
+    <tr>
+      <th>Fixed Lines in Location</th>
+      <td>Telefonica*<div></div></td>
+      <td>
+        <a href="https://github.com/camaraproject/APIBacklog/pull/92">Template</a><div>2024/10/01</div>
+      </td>
+      <td><ul><li>TSC Approved (2024/10/17)</li></ul></td>
+      <td>Pending</td>
+      <td>N/A</td>
+    </tr>
+    <tr>
+      <th>Line Eligibility for Carrier Billing API</th>
+      <td>Telefonica*<div></div></td>
+      <td>
+        <a href="https://github.com/camaraproject/APIBacklog/pull/94">Template</a><div>2024/10/01</div>
+      </td>
+      <td><ul><li>TSC Approved (2024/10/28)</li></ul></td>

```

Comment:
See previous comments ... the proposals are not even merged yet in APIBacklog repository.